recipes/pytest-ignore-flaky/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "pytest-ignore-flaky" %}
+{% set version = "0.1.1" %}
+{% set sha256 = "78f8ddf9b405c09ce852a4ceac58258e4b8d98d2bc236a03d4a075cdc51c56c6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: pip install .
```

Comment:
I think if you're going to use `pip` here it should be:

```
script: pip install --no-deps .
```

or possibly:

```
python -m pip install --no-deps --ignore-installed .
```

see:
https://github.com/conda-forge/staged-recipes/issues/528